project.js

```diff
@@ -472,6 +472,20 @@ function setupObjectShaderBuffers(obj) {
     );
 }
 
+function getTimeAngle() {
```

Comment:
incorporate the minutes. 24 hours * 60 minutes = 1440 minutes. so the index should be time of day in minutes (hrs * 60 + minutes) / 1440 * 360. Index = Int(((hr*60 + mins) / 1440) * 360)